### PR TITLE
Auth with keycloak

### PR DESCRIPTION
**Pull request by Cerfoglg**
Added authentication against keycloak, expanded configuration env variables in Dockerfile

### CODE REVIEW

File: config/config.yml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 default: &default
   config:
-    secret_key_base: 7a878b41eb072374207f3141d33743b3a7a03ac85cc22308c5923c38c7422e9e8b04ed2ae75f110df27e7f9e4b2b8c639cef0dde09ba7ca482ebca55668fdd15
-    registerable: true
+    secret_key_base: c684380abad95b8323c9970f31e90f2a8396560ed8b3242ebdbfe18dbef9e14928b10c1b102fb57608644d9c64b414f6e9d20a8f84002b4c9f725348176bcbd8
```
Comment (taliaga) [inline config/config.yml:5]: quick question, where was this coming from again?

File: config/config.yml [hunk 2]
```diff
@@ -17,7 +17,8 @@ default: &default
       authentication: plain
       enable_starttls_auto: false
     system_mailer_receiver: noreply@beacon-os.com
-    create_test_app_on_new_account: true
+    redis_url: redis://localhost:6379
```
Comment (taliaga) [inline config/config.yml:20]: this will be overridden by the env variable config, right?

**Review by taliaga — APPROVED** [review]

File: config/config.yml (hunk 1 above)
Comment (Cerfoglg) [reply config/config.yml:5]: @taliaga It's randomly generated

File: config/config.yml (hunk 2 above)
Comment (Cerfoglg) [reply config/config.yml:20]: @taliaga Yes

**Pull request merged**